torch/export/exported_program.py

```diff
@@ -350,6 +350,7 @@ def _decompose_and_get_gm_with_new_signature_constants(
 
         fake_args_unwrapped = pytree.tree_unflatten(fake_args, mod._in_spec)
         fake_mode = detect_fake_mode(fake_args)
+        fake_mode = contextlib.nullcontext() if fake_mode is None else fake_mode
```

Comment:
Can we use_detect_fake_mode_from_gm? Compared with plain detect_fake_mode, it also detect fake modes based onmeta["val"] for intermediate results. Maybe fake_mode won't be None after using it.